realm/realm-library/src/androidTest/java/io/realm/RealmAsyncQueryTests.java

```diff
@@ -113,6 +114,79 @@ public void onError(Exception e) {
         exitOrThrow(executorService, signalCallbackFinished, signalClosedRealm, backgroundLooper, threadAssertionError);
     }
 
+    public void testAsyncTransactionThatThrowsRuntimeException() throws Throwable {
+        final TestHelper.TestLogger testLogger = new TestHelper.TestLogger();
+        RealmLog.add(testLogger);
```

Comment:
You also need to remove it again, since this is a static field and not doing so can impact other tests.